tests/MultipleAuthSchemes/Startup.cs

```diff
@@ -44,8 +44,8 @@ public void ConfigureServices(IServiceCollection services)
                     .AddDownstreamWebApi("DownstreamB2CApi", Configuration.GetSection("DownstreamB2CApi"))
                     .AddInMemoryTokenCaches();
 
-            services.Configure<MicrosoftIdentityOptions>(OpenIdConnectDefaults.AuthenticationScheme,
-                                             o => o.ClientSecret = "");
+            //services.Configure<MicrosoftIdentityOptions>(OpenIdConnectDefaults.AuthenticationScheme,
```

Comment:
Yes, we want to remove these lines completely :)

Reply:
i might leave, as it's the dev app, in case we want to test manually like this.